app/__init__.py

```diff
@@ -3,69 +3,72 @@
 app = Flask(__name__)
 
 
-requests = {}
+requests = []
 
-users = {}
+users = []
 
 
 # POST a user
-@app.route('/users', methods=['POST'])
+@app.route('/api/v1/users', methods=['POST'])
 def create_user():
     user = {
         "username": request.json.get("username"),
         "email": request.json.get('email'),
         "password": request.json.get("password"),
-        "cconfirm_password": request.json.get('cconfirm_password'),
+        "confirm_password": request.json.get('confirm_password')
```

Comment:
Confirm password should not be in the user object, the id should be in the object. Your create user function should have a way to check if the user with the username or email already exists before it is appended. Write tests to check those edge cases.